deploy/lib/parca-agent/parca-agent.libsonnet

```diff
@@ -22,6 +22,9 @@ local defaults = {
   debugInfoUploadDisable: false,
   socketPath: '',
 
+  hostDbusSystem: false,
```

Comment:
So it will be disabled by default.

It was the best effort previously, maybe we can do the same and enable it always and let it silently fail if can't communicate with systemd dbus. What do you think?

Reply:
We can give it a try, I am always worried about what happens when the source does not exist (it's probably edge cases tho like the socket does not exist, the mount happens and creates a directory, D-Bus tries to start and wants to create its socket... I guess in most cases D-Bus should have started long before or it means it is not there)